### PR TITLE
Add READONLY and READWRITE commands

### PR DESCRIPTION
**Pull request by theodesp**
### Pull Request check-list

_Please make sure to review and check all of these items:_

- [x] Does `$ python setup test` pass with this change (including linting)?
- [x] Does travis tests pass with this change (enable it first in your forked repo and wait for the travis build to finish)?
- [x] Is the new or changed code fully tested?
- [ ] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?

_NOTE: these things are not required to open a PR and can be done
afterwards / while the PR is open._

### Description of change
Fixes https://github.com/andymccurdy/redis-py/issues/1050
_Please provide a description of the change here._
Added 2 commands READONLY and READWRITE that are relevant to the cluster context

### CODE REVIEW

File: tests/test_commands.py [hunk 1]
```diff
@@ -1639,6 +1639,19 @@ def test_cluster_slaves(self, mock_cluster_resp_slaves):
         assert isinstance(mock_cluster_resp_slaves.cluster(
             'slaves', 'nodeid'), dict)
 
+    @skip_if_server_version_lt('3.0.0')
+    def test_readwrite(self, r):
+        assert bool_ok(r.readwrite())
```
Comment (andymccurdy) [inline tests/test_commands.py]: Why can we not just use `assert r.readwrite()`? Shouldn't the `bool_ok` response callback have already been called?

Comment (theodesp) [reply tests/test_commands.py]: @andymccurdy Sorry for the delay. PR updated

**Comment by gyermolenko** [comment]
seems like a glitch from Travis. Maybe job should be restarted.

**Comment by andymccurdy** [comment]
I took care of the conflicts and merged this. Thanks!

**Pull request closed without merging**